File: app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id          :integer          not null, primary key
#  flight_name :string
#  origin      :string
#  destination :string
#  flight_date :datetime
#  airplane_id :integer
#  created_at  :datetime         not null
#  updated_at  :datetime         not null
#

class Flight < ActiveRecord::Base
end

File: app/assets/javascripts/models/flight.js
var app = app || {};

app.Flight = Backbone.Model.extend({
  // urlRoot: function () {
  //   return '/airplanes/' + this.get('airplane_id') + '/flights';
  // },
  defaults: {
    flightName: '',
    origin: '',
    destination: '',
    flightDate: '',
    airplaneID: ''
  }
});